tests/integration/fixtures.py

```diff
@@ -373,10 +373,10 @@ def _create_parameter(**kwargs):
         secretsmanager_client.delete_secret(SecretId=item)
 
 
-only_localstack = pytest.mark.skipif(
-    os.environ.get("TEST_TARGET") == "AWS_CLOUD",
-    reason="test only applicable if run against localstack",
-)
+@pytest.fixture
```

Comment:
Out of curiosity - did we make this change to allow dynamically assigning a value to `os.environ["TEST_TARGET"]` during test execution? I kind of liked the decorator style `@only_localstack` - makes the condition a bit more explicit. 

Looks like `skipif` also allows to specify a condition string, e.g. `pytest.mark.skipif('os.environ.get("TEST_TARGET") == "AWS_CLOUD"')` - could that be an option? (not sure if that gets lazily evaluated at runtime right before the execution of the annotated test method starts, though..)

Reply:
I made this change as to use the decorator, you have to add the import.
```
from tests.integration.fixtures import only_localstack
```
we import from fixtures, so I decided to make it a fixture, so no need for additional import :)

Using fixtures is always less explicit, you should know existing fixtures.

I can revert this change, also I could rename the fixture to `skipif_aws_cloud` to make it more explicit by name.

cc @dominikschubert

Reply:
The proper way here is probably to use neither of these, though if we use one, I'd personally prefer the previous one (the annotation).

A better approach that also avoids the fixtures import statement would be marking these tests with a custom marker =>
```
@pytest.mark.only_localstack
```
We could also think about a better name in this case for tests that are incompatible with the cloud provider, but for now it's probably fine.

The actual skipping should then be done in conftest.py by filtering these marks and exluding them from the test run if we target AWS_CLOUD.

Reply:
I will revert my change for now, and we can address this in separate PR.